cookbook/core/control_flow/dynamics.py

```diff
@@ -5,9 +5,23 @@
 A workflow is typically static where the directed acyclic graph's (DAG) structure is known at compile-time. However,
 scenarios exist where a run-time parameter (e.g. the output of an earlier task) determines the full DAG structure.
 
-Dynamic workflows can be used in such cases. Here's a code example that counts the common characters between any two
-strings.
+Dynamic workflows can be used in such cases.
```

Comment:
consider merging with previous sentence:
``
However, in cases where a run-time parameter (e.g. the output of an earlier task) determines the full DAG structure, you can use dynamic workflows by decorating a function with ``@dynamic``.
``